src/app/shared/components/template/components/map/map.component.scss

```diff
@@ -0,0 +1,14 @@
+.map-container {
+  width: 100%;
+}
+
+.map {
+  width: 100%;
+  height: 400px;
```

Comment:
nit(non-blocking): does the map container need a fixed height or can we make it more flexible? Can imagine cases where we might want the map to take up the majority of the screen.

Also just to check, does the map intercept touch events? If so you might either want to limit the width or add some sort of toggle if there is content above/below the map as it might not be possible to scroll the screen if the map is taking up the full view